### PR TITLE
I added German translations to Veil testmod

### PR DESCRIPTION
**Pull request by IDoTheHax**
Ich habe deutsche Übersetzungen hinzugefügt, sehr gut, sehr gut

feel free to edit

### CODE REVIEW

**Comment by Redfan2** [comment]
I‘d suggest waiting with merging this so that people that may want to review it get the time to do so

**Comment by IDoTheHax** [comment]
yes i agree im also gonna add another one in a few mins

**Comment by IDoTheHax** [comment]
Added dutch translations shall i go for danish or swedish next?

File: common/src/main/resources/assets/veil/lang/de_de.json [hunk 1]
```diff
@@ -0,0 +1,178 @@
+{
+  "key.veil.editor": "Editor umschalten",
+  "key.categories.veil": "Schleier",
```
Comment (shimizu-izumi) [inline common/src/main/resources/assets/veil/lang/de_de.json]: The mod name is identical to the category name and therefore shouldn't be translated.

**Review by shimizu-izumi — CHANGES_REQUESTED** [review]

**Comment by shimizu-izumi** [comment]
Most things look good to me as a native, but from my personal experience, the names of the shader types are typically referred to by their English names which might be something for consideration to change. Overall it is good enough.

**Review by shimizu-izumi — APPROVED** [review]
The German translation looks good to me as a native, but from my personal experience, the names of the shader types are typically referred to by their English names which might be something for consideration to change.

**Pull request merged**